src/styles/blocks/button.scss

```diff
@@ -0,0 +1,34 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  box-sizing: border-box;
+  width: 100%;
+  height: 56px;
+  border-radius: 40px;
+
+  text-decoration: none;
+  color: $c-main;
+  font-weight: 700;
+  line-height: 27px;
+  background-color: #fff;
+
+  transition: all 0.3s;
```

Comment:
better use `transition: [value] [duration]`, not `all`, because spmetimes it can work incorrect